examples/token/out/acts/Token_transfer.act

```diff
@@ -0,0 +1,20 @@
+behaviour transfer of Token
```

Comment:
lets remove the `examples/<name>/out` directory from git - this should be generated by the user